idseq_dag/steps/run_star.py

```diff
@@ -5,17 +5,17 @@
 import idseq_dag.util.command as command
 import idseq_dag.util.log as log
 import idseq_dag.util.s3 as s3
-
+import idseq_dag.util.count as count
 
 class PipelineStepRunStar(PipelineStep):
     def run(self):
         """Run STAR to filter out host reads."""
         # Setup
         input_files = self.input_files_local[0][0:2]
-        num_inputs = len(self.input_files[0])
+        num_inputs = len(input_files)
         scratch_dir = os.path.join(self.output_dir_local, "scratch_star")
 
-        total_counts_from_star = {}
+        self.total_counts_from_star = {}
```

Comment:
I think this variable should be initialized in __init__  for consistency reasons.